rust/Cargo.toml

```
@@ -30,19 +30,27 @@ http = "1.1.0"
 lazy_static = "1.4.0"
 log = "0.4.17"
 mime = "0.3.17"
-pyo3 = { version = "0.23.5", features = [
+pyo3 = { version = "0.24.2", features = [
     "macros",
     "anyhow",
     "abi3",
     "abi3-py39",
 ] }
-pyo3-log = "0.12.0"
-pythonize = "0.23.0"
+pyo3-log = "0.12.3"
+pythonize = "0.24.0"
 regex = "1.6.0"
 sha2 = "0.10.8"
 serde = { version = "1.0.144", features = ["derive"] }
 serde_json = "1.0.85"
 ulid = "1.1.2"
+reqwest = { version = "0.12.15", default-features = false, features = [
+    "http2",
+    "stream",
+    "rustls-tls-native-roots",
+] }
+http-body-util = "0.1.3"
+futures = "0.3.31"
+tokio = { version = "1.38.2", features = ["rt", "rt-multi-thread"] }
```

Unless there is a particular reason for not using the latest version?
```suggestion
tokio = { version = "1.44.2", features = ["rt", "rt-multi-thread"] }
```

---

Hmm, I assumed I just did `cargo add` :/